include/deal.II/base/tensor.h

```diff
@@ -1499,20 +1465,54 @@ operator- (const Tensor<rank,dim,Number> &p, const Tensor<rank,dim,OtherNumber>
  */
 //@{
 
+
 /**
- * Returns the contraction of two Tensors of rank 0.
+ * The dot product (single contraction) for tensors: Return a tensor of
+ * rank $(\text{rank\_1} + \text{rank\_2} - 2)$ that is the contraction of
+ * the last index of a tensor @p src1 of rank @p rank_1 with the first
+ * index of a tensor @p src2 of rank @p rank_2:
+ * @f[
+ *   \text{result}_{i_1,..,i_{r1},j_1,..,j_{r2}}
+ *   = \sum_{k}
+ *     \text{left}_{i_1,..,i_{r1}, k}
+ *     \text{right}_{j_1,..,j_{r2}, k}
+ * @f]
+ *
+ * @note For the <tt>Tensor</tt> class, the multiplication operator only
+ * performs a contraction over a single pair of indices. This is in
+ * contrast to the multiplication operator for symmetric tensors, which
```

Comment:
say `SymmetricTensor` to make clear that you're talking about the class, not any `Tensor` object that happens to be symmetric.